### PR TITLE
fix: placeholders

### PR DESCRIPTION
**Pull request by fkakatie**
## Description

* fetch localized `placeholders.json`
* use localized placeholders as `alt` text for `img`s, `title` text for `svg`s

## Related Issue

- Icon alt texts 287
- Introduce placeholders 307

## Links

- https://placeholders--express-website--adobe.hlx3.page/express/
- https://placeholders--express-website--adobe.hlx3.page/express/pricing
- https://placeholders--express-website--adobe.hlx3.page/dk/express/pricing
- https://placeholders--express-website--adobe.hlx3.page/tw/express/pricing

### CODE REVIEW

**Comment by fkakatie** [comment]
beginnings of placeholder functionality... because the icon building happens so early, i wonder if it makes more sense to "fix" the icons, then update the alt text later, maybe with the lazy or delayed scripts? what do you think?

File: express/scripts/scripts.js [hunk 1]
```diff
@@ -1010,7 +1035,8 @@ export function fixIcons(block = document) {
           // console.log(blockName);
           if (smallIconBlocks.includes(blockName)) size = 22;
         }
-        $picture.parentElement.replaceChild(getIconElement([icon, mobileIcon], size), $picture);
+        $picture.parentElement
+          .replaceChild(getIconElement([icon, mobileIcon], size, placeholders), $picture);
```
Comment (rofe) [inline express/scripts/scripts.js:1039]:
```js
          .replaceChild(getIconElement([icon, mobileIcon], size, placeholders[icon]), $picture);
```

**Review by rofe — CHANGES_REQUESTED** [review]

File: express/scripts/scripts.js [hunk 2]
```diff
@@ -737,6 +743,25 @@ export function getMetadata(name) {
...
+ * fetches the string variables.
+ * @returns {object} localized variables
+ */
+
+export async function fetchPlaceholders() {
+  if (!window.placeholders) {
+    const locale = getLocale(window.location);
+    const urlPrefix = locale === 'us' ? '' : `/${locale}`;
+    const resp = await fetch(`${urlPrefix}/express/placeholders.json`);
+    const json = await resp.json();
```
Comment (rofe) [inline express/scripts/scripts.js:756]: I think we should guard for `404s` here (and potentially fall back to english in this case?)

**Review by rofe — CHANGES_REQUESTED** [review]

File: express/scripts/scripts.js (hunk 2 above)
Comment (fkakatie) [reply express/scripts/scripts.js]: i _think_ this may be resolved, but would appreciate your feedback.

**Review by rofe — APPROVED** [review]
LGTM, except: `fetch` does not throw on `404`, but `resp.ok` will be `false` and `resp.status` will be set. So I'd change the fallback logic from catching an error to checking `resp.ok`.

**Comment by rofe** [comment]
Merging now, we can add the English fallback later - if needed

**Pull request merged**